packages/hash/frontend/src/components/layout/PageHeader/ActionsDropdown.tsx

```diff
@@ -56,16 +56,23 @@ export const ActionsDropdown: React.FC<{
       <IconButton
         size="medium"
         rounded
-        sx={{
+        sx={({ palette }) => ({
           mr: {
             xs: 1,
             md: 1.5,
           },
-          color: open ? theme.palette.common.white : theme.palette.gray[40],
-          backgroundColor: open
-            ? theme.palette.blue["70"]
-            : theme.palette.gray[20],
-        }}
+          color: open ? palette.common.white : palette.gray[40],
+          backgroundColor: open ? palette.blue["70"] : palette.gray[20],
+
+          ":hover": {
```

Comment:
Nit: spotted the icon has the wrong gray color on hover. Currently it's `gray[80]`, whereas the designs say it's `gray[50]`